spec/decorators/provider_decorator_spec.rb

```diff
@@ -1,6 +1,26 @@
 require "rails_helper"
 
 RSpec.describe ProviderDecorator do
+  describe "#partner_provider_placements" do
+    it "returns placements belonging to a partner provider" do
+      provider = create(:provider)
+      subject = create(:subject)
+      region = Region.create!(name: "Springfield")
+      school = Placements::School.create!(
+        name: "Springfield Elementary",
+        urn: "1234567890",
+        region:,
+      )
+      placement = Placement.create!(provider:, school:, subject:)
+
+      decorated_provider = provider.decorate
+      result = decorated_provider.partner_provider_placements(school)
+      decorated_placement = placement.decorate
+
+      expect(result).to include(decorated_placement)
+    end
```

Comment:
Could we check the inverse of this is also true please? Placements that do not belong to the partner provider are not shown?